### PR TITLE
fix AddToScheme to add all served versions

### PR DESCRIPTION
**Pull request by RamLavi**
**What this PR does / why we need it**:
We are currently using the old shceme version when
adding resources to schema.
Let's change it to use all served versions v1 and v1alpha1

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/apis/addtoscheme_networkaddonsoperator_v1alpha1.go [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package apis
+
+import (
+	cnaov1alpha1 "github.com/kubevirt/cluster-network-addons-operator/pkg/apis/networkaddonsoperator/v1alpha1"
+)
+
+func init() {
+	// Register the types with the Scheme so the components can map objects to GroupVersionKinds and back
+	AddToSchemes = append(AddToSchemes, cnaov1alpha1.SchemeBuilder.AddToScheme)
```
Comment (qinqon) [inline pkg/apis/addtoscheme_networkaddonsoperator_v1alpha1.go:9]: This is raceful in case `init` is not sequencially called.

**Review by qinqon — COMMENTED** [review]
Question related to golang "init" function

File: pkg/apis/addtoscheme_networkaddonsoperator_v1alpha1.go (hunk 1 above)
Comment (RamLavi) [reply pkg/apis/addtoscheme_networkaddonsoperator_v1alpha1.go:9]: well it's used in the same manner in knmstate and KMP, but I didn't find solid evidence that it runs sequentially, so let's be on the safe side.
Done

Comment (RamLavi) [reply pkg/apis/addtoscheme_networkaddonsoperator_v1alpha1.go:9]: it's done the same in knmstate

**Pull request merged**